Playbooks/playbook-Tanium_-_Ask_question.yml

```diff
@@ -0,0 +1,174 @@
+id: Tanium - Ask question
```

Comment:
when possible, switch all simple values to complex (using the filters & transformer)